Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/TimeStamper.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.util
+
+import org.joda.time.Instant
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class TimeStamper @Inject constructor() {
+
+    val nowUTC: Instant
+        get() = Instant.now()
```

Comment:
probably we should base this for 3 Hour Mode as well.

Reply:
True.

Don't need it now though, I just removed the `currentDate` being passed. The logic from previous code was incorrect when combined with the limit, it would the last 3 hours based on our current time, and not based on the date that we passed it, so if we didn't pass in `currentDate=today` then we would get some hours in the middle of the last day based on our current time.

But we don't need the currentDate, just a limit, i.e. `hourLimit` because we can just start from the newest date in the servers country index and go backwards until `hourLimit` is exhausted. :partying_face:

Leaving this open until you looked over my new changes regarding this.